astropy/table/index.py

```diff
@@ -606,24 +606,43 @@ def data(self):
         return self.index.data
 
 
-def get_index(table, table_copy):
-    '''
-    Inputs a table and some subset of its columns, and
-    returns an index corresponding to this subset or None
-    if no such index exists.
+def get_index(table, table_copy=None, names=None):
+    """
+    Inputs a table and some subset of its columns as table_copy or
+    dictionary containing  names of columns as names,and returns an index
```

Comment:
The `names` input is not a dictionary it is a list or tuple.

Reply:
This one is silly Sorry for it.